sentinel-core/src/main/java/com/alibaba/csp/sentinel/annotation/SentinelResource.java

```diff
@@ -77,6 +79,9 @@
      */
     String defaultFallback() default "";
 
+    Class<? extends IGlobalFallback> globalFallback() default DefaultGlobalFallback.class;
```

Comment:
Could you please add javadoc for it?

Reply:
Completed in the last commit

Reply:
> Could you please add javadoc for it?

Completed in the last commit

Reply:
> Could you please add javadoc for it?

A network error occurs during the CodeQL build, how to rebuild or resolve it